docs/python/python-tutorial.md

```diff
@@ -261,6 +261,34 @@ To install the required packages in your virtual environment, enter the followin
 
 Congrats on completing the Python tutorial! During the course of this tutorial, you learned how to create a Python project, create a virtual environment, run and debug your Python code, and install Python packages. Explore additional resources to learn how to get the most out of Python in Visual Studio Code!
```

Comment:
I would suggest to move this below the managing dependencies section and right above the "Next Steps" so that it follows the final steps in the tutorial.